### PR TITLE
added README.md and setup.py files

### PR DESCRIPTION
**Pull request by praneeth36**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,311 @@
...
+```
+
+### Types of handlers we support
+- nginx_access
+- django
+- mongodb
+- elasticsearch
+- basescript
+
+### Getting help per command
```
Comment (venugopalkandru) [inline README.md]: @praneeth36 @AnushaKonduru We can how something like `Help command` as this section title.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,311 @@
...
+  --mongodb-password MONGODB_PASSWORD
+                        password to authenticate mongodb
+  --mongodb-database MONGODB_DATABASE
+                        database to store logs
+  --mongodb-collection MONGODB_COLLECTION
+                        collection to store logs
+
+```
+
+### Run the `collector`
```
Comment (venugopalkandru) [inline README.md]: @praneeth36 @AnushaKonduru We should have this section title as `How to run log collector?`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,311 @@
...
+```bash
+logagg collect /path/to/input/log_file:logagg.collect.handlers.<handler_name> <topic name> --nsqd-http-address <nsqd http address>
+```
+
+- Example run command:
+```bash
+logagg collect /var/log/nginx/access.log:logagg.collect.handlers.nginx_access nginx --nsqd-http-address localhost:4151  
+```
+
+### Run the `forwarder`
```
Comment (venugopalkandru) [inline README.md]: @praneeth36 @AnushaKonduru `How to run log forwarder?` can be the title here.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,311 @@
...
+logagg forward --nsqtopic nginx --nsqchannel test --nsqd-tcp-address localhost:4150 --mongodb-user-name abc --mongodb-password xxxxxx --mongodb-server-url localhost:27017 --mongodb-port 27017 --mongodb-database logs --mongodb-collection nginx
+```
+
+### Check the message traffic at nsq
+- We can check, how many messages that are being written to nsq and reading from nsq through the browser, by going through the link:
+```bash
+localhost:4171
+```
+
+### How to check the records at forwarder end.
```
Comment (venugopalkandru) [inline README.md]: @praneeth36 @AnushaKonduru this should end with a `?`.

**Comment by AnushaKonduru** [comment]
@venugopalkandru 

* We pass the README through Grammarly, modified the README and given pull request. Thank you.

**Pull request merged**